db/migrate/20191002232744_add_user_id_to_adjustments.rb

```diff
@@ -0,0 +1,14 @@
+class AddUserIdToAdjustments < ActiveRecord::Migration[5.2]
+  def up
+    add_reference :adjustments, :user, foreign_key: true
+
+    Adjustment.all.each do |adjustment|
+      adjustment.user_id = adjustment.organization.users.find_by(organization_admin: true).id
+      adjustment.save
```

Comment:
Great! Yeah -- this is what I was asking for, so thank you. 

Fun trivia (not a blocker):
You can do this in one step with the `update` method:
```ruby
adjustment.update(user: adjustment.organization.users.find_by(organization_admin: true))
```